### ISSUE TITLE
[iOS] _visualEffectView is null, or: BlurViewManager.m vs BlurView.m

### ISSUE THREAD

**Issue by adgarcia** [react-native-community/react-native-blur#97]
I think I'm missing something important because the library, as written, doesn't actually do anything when I install + link + add the BlurView component. 

Here's my render function: 

``` javascript
render() {
    const image = 'http://iphonewallpapers-hd.com/thumbs/firework_iphone_wallpaper_5-t2.jpg';
    return (
      <Image source={{uri: image}} style={styles.container}>
        <BlurView style={{flex: 1, alignSelf: 'stretch', backgroundColor: 'transparent', justifyContent: 'center'}}>

        </BlurView>

      </Image>
    );
  }
```

On the simulator (running ios 9.3), this doesn't do anything. I have only a blurless image =(

I poked around in the native code, and found that BlurView.m isn't actually the module that's exported; that would be BlurViewManager.m

BlurView is pulled in here:

``` javascript
import React, { Component, PropTypes } from 'react';
import { requireNativeComponent } from 'react-native';

class BlurView extends Component {
  render() {
    return (
      <NativeBlurView
        {...this.props}
        style={[{
          backgroundColor: 'transparent',
        }, this.props.style
        ]}
      />
    );
  }
}

BlurView.propTypes = {
  blurType: PropTypes.string,
};

const NativeBlurView = requireNativeComponent('BlurView', BlurView);

module.exports = BlurView;
```

But the only file with a module export is BlurViewManager:

``` objective-c
#import "BlurViewManager.h"
#import "BlurView.h"

@implementation BlurViewManager

RCT_EXPORT_MODULE();

- (UIView *)view
{
    return [[BlurView alloc] init];
}

RCT_EXPORT_VIEW_PROPERTY(blurType, NSString);

@end
```
# tl:dr

the function `-setBlurType` is never getting called, which means that `_visualEffectView` is null in function `-layoutSubviews` of class BlurView. Everything starts to work correctly when I either
- initialize _visualEffectView in -layoutSubviews (if it's null)
  or
- pull in BlurViewManager instead of BlurView in the index.js file

Am I missing something obvious? If not, would it be all right if I submited a pull request?

**Comment by Kureev** [comment]
I just checked the newest version (2.0.0) with react-native@0.40 - can't reproduce the issue. Please, try to update react-native-blur library.
